pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. If you aren't using the list command, a new file called AUTHORS will be created. Sorted by amount of commits.
```

Comment:
```suggestion
> Display log or list of committers of a Git repository.
```
This doesn't really seem important.

Reply:
It does to me, since otherwise someone will be confused when a new random file appears in their directory

Reply:
How about that? (We should avoid `you`, `yours`, etc.)
```suggestion
> Display log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
```